terraform/deployments/govuk-publishing-infrastructure/security.tf

```diff
@@ -207,3 +207,73 @@ resource "aws_security_group_rule" "licensify_frontend_from_eks_workers" {
   security_group_id        = data.terraform_remote_state.infra_security_groups.outputs.sg_licensify-frontend_internal_lb_id
   source_security_group_id = data.terraform_remote_state.cluster_infrastructure.outputs.node_security_group_id
 }
+
+resource "aws_security_group_rule" "ec2_www_origin_from_eks_workers" {
+  description       = "EC2 www-origin accepts requests from EKS NAT gateways"
+  type              = "ingress"
+  from_port         = 443
+  to_port           = 443
+  protocol          = "tcp"
+  security_group_id = data.terraform_remote_state.infra_security_groups.outputs.sg_cache_external_elb_id
```

Comment:
I have a sneaky suspicion that `sg_cache_external_elb_id` might be the confusing disused/left-over one from the early days of the AWS migration. Definitely check this out if you haven't already. My money's on that it turns out we want `cache_public_lb_id` instead :)

Reply:
Ohh hang on, I think I see what I might have missed here. I bet it's that the security group still bears the name of the old "external" LB but refers to the new "public" one.

If you've tested it and it works, I must be missing something 😅